tests/test_tqdm_utils.py

```diff
@@ -0,0 +1,22 @@
+from unittest.mock import patch
+
+from transformers import AutoModel
+from transformers.testing_utils import require_torch
+from transformers.utils.tqdm_utils import set_progress_bar_enabled
+
+
+TINY_MODEL = "hf-internal-testing/tiny-random-distilbert"
+
+
+@require_torch
+def test_set_progress_bar_enabled():
+    with patch("tqdm.auto.tqdm") as mock_tqdm:
+        set_progress_bar_enabled(True)
+        _ = AutoModel.from_pretrained(TINY_MODEL, force_download=True)
+        mock_tqdm.assert_called()
+
+        mock_tqdm.reset_mock()
+
+        set_progress_bar_enabled(False)
+        _ = AutoModel.from_pretrained(TINY_MODEL, force_download=True)
+        mock_tqdm.assert_not_called()
```

Comment:
Could this download a configuration file instead, to be framework agnostic?

And this test could go into `tests/test_logging.py` so as to not add a new file.

Reply:
and then you can probably remove `require_torch`, once switched to `AutoConfig.from_pretrained(...)`

Reply:
To keep in line with the structure of the test directory (i.e. having `tqdm` tests in `test_logging,py`), does it also make sense to move `tqdm`-related code to `utils/logging.py` instead of keeping them in `utils/tqdm_utils.py`?

Reply:
I think it's a great idea and consistent with logging tests and docs.

But perhaps secure another agreement from @sgugger or @LysandreJik before you act on it.

Reply:
I made this a tentative item on the to-do, but I'll wait for more feedback!

Reply:
Yes, this sounds good to me! Having everything in `utils/logging.py` sounds great.